actions/channel_actions.jsx

```diff
@@ -104,6 +106,9 @@ export function executeCommand(message, args, success, error) {
     case '/settings':
         GlobalActions.showAccountSettingsModal();
         return;
+    case '/collapse':
+    case '/expand':
+        actionOnItemsWithPrefix(StoragePrefixes.embedVisible, () => null)(dispatch, getState);
```

Comment:
I am setting values of all keys with prefixes to `null`. Should I delete them instead? Let me know if there is an easy way to delete keys with existing actions. I tried couple of things but they seemed more like hacks. Adding an action/reducer like `deleteActionOnItemsWithPrefix` is the only way I see other than this